src/com/company/Sorting.java

```diff
@@ -3,47 +3,51 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
-import static com.company.Main.*;
 
 public class Sorting {
 
     public static int sortMethodNumber;
-    // to select sorting method
-    public static void selectSortingMethod(){
+    public static List<Integer> numbersToSave;
 
-        System.out.println ("\n Select sorting method by" + "\n" +
+    public static void selectSortingMethod() {// to select sorting method
+
+        System.out.println("\n Select sorting method by" + "\n" +
                 "selecting corresponding number: " + "\n" + "\n" +
                 "Bubble sorting - press key '1' " + "\n" +
                 "Sorting by inserts - press key '2' " + "\n" +
                 "Sorting by choice - press key '3' " + "\n"
         );
 
-        Scanner scanSelect = new Scanner(System.in);
-
-        if (scanSelect.hasNext()){
-            sortMethodNumber = scanSelect.nextInt();
+        try {
+            Main.scanner = new Scanner(System.in);
```

Comment:
Sorrting doesn't know about Main file.

Reply:
Fixed. no reference from Sorting to Main